src/Core/Services/GraphQLSchemaCreator.cs

```diff
@@ -60,6 +61,18 @@ public class GraphQLSchemaCreator
         {
             RuntimeConfig runtimeConfig = runtimeConfigProvider.GetConfig();
 
+            if (runtimeConfig.Runtime is not null &&
```

Comment:
Can all this code can be moved to `RuntimeConfig` (or any other appropriate place) and reused wherever required?

Reply:
Moved this logic to a helper method within `RuntimeConfig`